packages/codeeditor/src/lineCol.tsx

```diff
@@ -125,15 +120,19 @@ class LineFormComponent extends React.Component<
                 this._textInput = input;
               }}
             />
-            <div className={lineFormButtonDiv}>
+            <div className="jp-baseLineForm jp-lineFormButtonDiv">
```

Comment:
**Nit**: Use container or wrapper instead of a tag name div in the class name.

Reply:
:+1: